JoystickProcess.py

```diff
@@ -0,0 +1,26 @@
+import time
+import os
+import pygame
+from threading import Thread, BoundedSemaphore
+from robocluster import device
+
+LOOP_PERIOD = 100
```

Comment:
In Roveberrypy this was miliseconds, but in robocluster, `device.every()` interprets numerical literals as seconds.